packages/firestore/src/local/indexeddb_schema.ts

```diff
@@ -350,6 +346,71 @@ export class SchemaConverter implements SimpleDbSchemaConverter {
   }
 }
 
+/**
+ * Delete a mutation batch and the associated document mutations.
+ * @returns A PersistencePromise of the document mutations that were removed.
+ */
+export function removeMutationBatch(
```

Comment:
This doesn't seem like it belongs logically with indexedb_schema.

Would it make sense to have something like a mutation_batch.ts that holds this and related functions?

Reply:
Moved to indexeddb_mutation_batch_impl.ts